packages/builder/src/components/design/NavigationPanel/ScreenWizard.svelte

```diff
@@ -32,17 +110,12 @@
 </script>
 
 <Modal bind:this={newScreenModal}>
-  <NewScreenModal bind:selectedScreens {chooseModal} />
+  <NewScreenModal bind:selectedScreens {save} {chooseModal} />
 </Modal>
 
 <Modal bind:this={screenDetailsModal}>
-  <ScreenDetailsModal bind:screenName bind:url {chooseModal} />
+  <ScreenDetailsModal bind:screenName bind:url {save} {chooseModal} />
 </Modal>
 <Modal bind:this={navigationSelectionModal}>
-  <NavigationSelectionModal
-    bind:url
-    bind:screenName
-    bind:selectedScreens
-    {chooseModal}
-  />
+  <NavigationSelectionModal {chooseModal} {save} />
```

Comment:
We may as well just fully remove this code that references `NavigationSelectionModal` at the moment since if we include it like this then the file will still be bundled with the builder even though it isn't needed.